examples/schemas/month.json

```diff
@@ -29,9 +29,14 @@
 				},
 				"pause": {
 					"$ref": "#/definitions/time_span"
+				},
+				"vacation": {
+					"type": "boolean",
+					"default": false
 				}
 			},
 			"required": ["action", "day", "start", "end"],
+			"not": { "required": ["pause", "vacation"] },
```

Comment:
Does this mean that only one of these values may be present? If so why not using oneOf? I read something about possible misbehaviour here https://stackoverflow.com/questions/24023536/how-do-i-require-one-field-or-another-or-one-of-two-others-but-not-all-of-them. Could that be a problem?

Reply:
If I understand this correctly, then `oneOf` means exactly one of the specified schemas, whereas `not required` means not all of the schemas together, in this case one or none of the schemas. So there is a difference, but I agree that it is confusing. Maybe `not allOf` is more readable, but I will check to ensure the behaviour is correct.

Reply:
Now that I think about it, we're not specifying schemas here, but properties, that's why we need `require` here.

Reply:
Could you check [https://json-schema.org/understanding-json-schema](https://json-schema.org/understanding-json-schema/index.html) and see if I understood things right?

Reply:
As far as I can see, https://json-schema.org/understanding-json-schema/reference/combining.html?highlight=oneof confirms your statement concerning the combination of subschemes and properties. Therefore, I'm now convinced that your solution does what it is meant for :)